docs/releases/dev.rst

```diff
@@ -34,6 +34,10 @@ Details of changes
   is now built on the client. This requires refreshing all server stats using
   the :djadmin:`refresh_stats_rq` command (:issue:`3835`).
 
+- Disabled projects are visually differentiated in the projects drop-down
+  (:issue:`3996`). Since the in-cache data structure supporting this changed,
+  it's necessary to clear the cache.
```

Comment:
> Perhaps it is a good idea to provide some hint on how to do that.

Any specific user-friendly suggestions? if you read the PR description I'm asking for help on that.